Vagrantfile

```diff
@@ -437,7 +437,14 @@ Vagrant.configure('2') do |config|
     end
   end
 
-  config.vm.hostname = 'vvv'
+  # Enable hostnames to be set via vm_config.hostname, but default to "vvv".
+  if defined? vvv_config['vm_config']['hostname'] then
```

Comment:
This should be set using a default value further up the file so it's always defined